aisdc/attacks/likelihood_attack.py

```diff
@@ -548,6 +622,21 @@ def main():
         default=P_THRESH,
         help=("Significance threshold for p-value comparisons. Default = %(default)f"),
     )
+
+    parser.add_argument(
+        "--shadow-models-fail-fast",
+        action="store",
+        type=parse_boolean_argument,
+        default=True,
```

Comment:
Instead of having a command line argument passed as `--shadow-models-fail-fast True` and `--shadow-models-fail-fast False` why not just have the default behaviour as `False` and then set the argument to `True` if `--shadow-models-fail-fast` is present?

Just need to change this to something like:
```python
    parser.add_argument(
        "--shadow-models-fail-fast",
        action="store_true",
        required=False,
        dest="shadow_models_fail_fast",
        help=(
            """To stop training shadow models early based on minimum number of
            confidences across all rows (--n-shadow-rows-confidences-min)
            in the shadow data."""
        ),
    )
```

Reply:
The default value is False when an argument is not passed.